Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2006-06-25
Initial Package Version: 1.4.2
Upstream Status: Not applicable
Origin: Gentoo
Description: Link to shared cxxlibs. Updated for 6u1.

--- jdk-build-orig/j2se/make/common/shared/Compiler-gcc.gmk	2007-02-10 15:22:33.000000000 -0600
+++ jdk-build/j2se/make/common/shared/Compiler-gcc.gmk	2007-02-09 17:04:06.000000000 -0600
@@ -45,7 +45,7 @@
   CC             = $(COMPILER_PATH)gcc
   CPP            = $(COMPILER_PATH)gcc -E
   # statically link libstdc++ before C++ ABI is stablized on Linux
-  STATIC_CXX     = true
+  STATIC_CXX     = false
   ifeq ($(STATIC_CXX),true)
     # g++ always dynamically links libstdc++, even we use "-Wl,-Bstatic -lstdc++"
     # We need to use gcc to statically link the C++ runtime. gcc and g++ use
